contrib/docs/tutorials/authenticate-api-requests.md

```diff
@@ -13,7 +13,7 @@ As techdocs HTML pages load assets without an Authorization header the code belo
 
 import cookieParser from 'cookie-parser';
 import { Request, Response, NextFunction } from 'express';
-import { JWT } from 'jose';
+import jwt_decode from 'jwt-decode';
```

Comment:
Could name this `jwtDecoder` instead, to get js casing (that's what we call it in our own code)